test/api/buffer/receivers/create/001-ok/expected-response.json

```diff
@@ -1,5 +1,5 @@
 {
   "receiverId": "github-pull-requests",
   "name": "Github Pull Requests",
-  "url": "https://buffer.connection.keboola.com/v1/import/github-pull-requests/#/%s"
+  "url": "https://%s.keboola.com/v1/import/github-pull-requests/#/%s"
```

Comment:
@jprochazk sorry, na toto som zabudol.
Spravna URL ma byt `buffer.keboola.com`.

Su 2 moznosti ako to ziskat:
- zo Storage API services, ale to jedivne, aby sa nasa servica pytal inej, aku ma URL
- cez ENV
  - https://github.com/keboola/keboola-as-code/blob/b0c0a9d4fa0d7fa017f514d0e2379b7a512e1d9e/provisioning/templates-api/kubernetes/templates/api/config-map.yaml#L10
   - https://github.com/keboola/keboola-as-code/blob/b0c0a9d4fa0d7fa017f514d0e2379b7a512e1d9e/provisioning/templates-api/kubernetes/templates/api/deployment.yaml#L43
   
   Tj. sprav prosim nejaku ENV, napr. `KBC_BUFFER_API_HOST`.

Reply:
Staci to vyriesit v kode + lokalny vyvoj, provisioning poriesim.

Reply:
Buffer API není per-stack?

Reply:
Je, a `keboola.com` je US stack. north-europe.azure.keboola.com/admin/auth/l

`<service>.<stack>`, napr. `connection.keboola.com`, `buffer.keboola.com`.

Reply:
Pozri https://help.keboola.com/overview/#stacks